examples/sampling-hamiltonian-mcmc.ipynb

```diff
@@ -681,21 +681,21 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "Python 2",
+   "display_name": "Python 3",
```

Comment:
Hey @lorcandelaney 

Testing is good, but try not to `git add` the notebooks every time you re-run them (it adds a lot of clutter to the commit history)